2_exe/calcuController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+class Calculator
```

Comment:
It is best practice to have the same naming convention for your class name an file name.

Please refer to PSR-1 and PSR-12 for changes.